### PR TITLE
Update the RichText style (bsc#1196296)

### PR DESCRIPTION
**Pull request by lslezak**
- Update also the style for the RichText *content* when changing the stylesheet (theme)
- https://bugzilla.suse.com/show_bug.cgi?id=1196296

### Screenshots

It fixes the visibility of the add-ons list in the registration step.

| The Original Behavior |  With the Fix|
|----|----|
| rich_text_broken | rich_text_fixed |

### CODE REVIEW

**Review by dgdavid — COMMENTED** [review]
Great! The result LGTM, but a code review from @shundhammer would be more than welcome :)

File: package/libyui.changes [hunk 1]
```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Thu Mar  3 07:57:59 UTC 2022 - Ladislav Slezák <lslezak@suse.cz>
+
+- Update also the stylesheet (theme) for the RichText content when
+  changing the UI theme (bsc#1196296)
```
Comment (dgdavid) [inline package/libyui.changes:5]: BTW, It fixes https://bugzilla.suse.com/show_bug.cgi?id=1196356 too, right?

**Review by dgdavid — APPROVED** [review]
@lslezak, @shundhammer, LGTM, thank you guys. Just bear in mind https://bugzilla.suse.com/show_bug.cgi?id=1196356, to see if it can be closed too.

**Comment by lslezak** [comment]
Already closed as a duplicate...

**Pull request merged**